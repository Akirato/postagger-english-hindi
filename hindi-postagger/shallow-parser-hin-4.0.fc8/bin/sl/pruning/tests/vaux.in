<Sentence id="1">
1	((	NP	
1.1	WodA	QF	<fs af='WodA,adv,,,,,,'>|<fs af='WodA,adj,m,sg,,d,,'>|<fs af='WodA,n,m,sg,3,d,0,0'>
1.2	sA	RP	<fs af='sA,psp,m,sg,,d,kA,kA'>
1.3	kAma	NN	<fs af='kAma,n,m,sg,3,d,0,0'>|<fs af='kAma,n,m,pl,3,d,0,0'>|<fs af='kAma,n,m,sg,3,o,0,0'>
	))		
2	((	VGF	
2.1	kara	VM	<fs af='kara,n,m,sg,3,d,0,0'>|<fs af='kara,n,m,pl,3,d,0,0'>|<fs af='kara,n,m,sg,3,o,0,0'>|<fs af='kara,v,any,any,any,,0,0'>
2.2	le	VAUX	<fs af='le,v,any,any,any,,0,0'>
	))		
3	((	NP	
3.1	waba	PRP	<fs af='waba,adv,,,,,,'>|<fs af='waba,avy,,,,,,'>
	))		
4	((	VGF	
4.1	AnA	VM	<fs af='AnA,n,m,sg,3,d,0,0'>|<fs af='A,v,m,sg,any,,nA,nA'>
4.2	.	SYM	<fs af='.,punc,,,,,,'>
	))		
</Sentence>

