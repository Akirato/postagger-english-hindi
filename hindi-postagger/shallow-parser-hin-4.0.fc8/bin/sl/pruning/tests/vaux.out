<Sentence id="1">
1	((	NP	
1.1	WodA	QF	<fs af='WodA,adj,m,sg,,d,,'>
1.2	sA	RP	<fs af='sA,psp,m,sg,,d,kA,kA' poslcat="NM">
1.3	kAma	NN	<fs af='kAma,n,m,sg,3,d,0,0'>|<fs af='kAma,n,m,pl,3,d,0,0'>|<fs af='kAma,n,m,sg,3,o,0,0'>
	))		
2	((	VGF	
2.1	kara	VM	<fs af='kara,v,any,any,any,,0,0'>
2.2	le	VAUX	<fs af='le,v,any,any,any,,0,0'>
	))		
3	((	NP	
3.1	waba	PRP	<fs af='waba,adv,,,,,,' poslcat="NM">|<fs af='waba,avy,,,,,,' poslcat="NM">
	))		
4	((	VGF	
4.1	AnA	VM	<fs af='A,v,m,sg,any,,nA,nA'>
4.2	.	SYM	<fs af='.,punc,,,,,,'>
	))		
</Sentence>
